pupil_src/shared_modules/pupil_producers.py

```diff
@@ -241,3 +245,16 @@ def init_ui(self):
                                     setter=lambda _: _)
         progress_slider.display_format = '%3.0f%%'
         self.menu.append(progress_slider)
+
+    @property
+    def detection_paused(self):
+        return self._detection_paused
+
+    @detection_paused.setter
+    def detection_paused(self, should_pause):
+        self._detection_paused = should_pause
+        for eye_id in range(2):
+            if self.eye_processes[eye_id] is not None:
+                self.notify_all({'subject': 'file_source.toggle_play',
```

Comment:
I prefer two notifications:

file_source.should_play
and 
file_source.should_stop